systemd/cbd-rrd.service

```diff
@@ -20,7 +20,7 @@
 Description=Centreon RRD Broker service
 
 [Service]
-EnvironmentFile=/etc/sysconfig/cbd_rrd
+EnvironmentFile=/etc/default/cbd_rrd
```

Comment:
But making these changes you break the build to the EL environment (rpm). The most correct is to make a sed (replace) in the debian rules file

Reply:
Yes, exactly!
Either we change it when it works or we stop using environment variables